chia/cmds/init_funcs.py

```diff
@@ -325,13 +333,15 @@ def chia_init(root_path: Path):
     if root_path.is_dir() and Path(root_path / "config" / "config.yaml").exists():
         # This is reached if CHIA_ROOT is set, or if user has run chia init twice
         # before a new update.
-        check_keys(root_path)
+        if not skip_check_keys:
```

Comment:
If possible, consider reversing the sense of `skip_check_keys` and renaming to `check_keys`

Reply:
I don't like the double-negative either. Let me ruminate on a better name.

Reply:
Renamed to `should_check_keys` and inverted the logic. `check_keys` as a parameter name conflicts with the function of the same name, and `checks_keys` is too similar I feel.